scripts/generate.js

```diff
@@ -26,7 +26,7 @@ function padding(length) {
 
 function generateReact(name) {
   return (
-`/*
+    `/*
```

Comment:
That looks odds...

Reply:
I'm assuming it's caused by `"indent": ["error", 2, { "SwitchCase": 1 }],`?

Reply:
I'm assuming it's caused by `"indent": ["error", 2, { "SwitchCase": 1 }],`? I can remove the new rules and tackle them in a different PR, if you would prefer

Reply:
Are these rules the same as for parse server?

Reply:
Yep!

Reply:
Then let's just ignore this for now I suggest